Cocoaheads School_Lesson2.playground/Contents.swift

```diff
@@ -52,7 +79,7 @@ class StackStatistics: Stack {
         }
     }
     
-    override func top() -> Int? {
+    override func top() -> Element? {
```

Comment:
```suggestion
    override func top() -> Element? {
```
And here please add a description for this method

Reply:
done